openmetadata-service/src/main/java/org/openmetadata/service/alerts/AlertsRuleEvaluator.java

```diff
@@ -225,4 +218,15 @@ public boolean matchAnyFieldChange(String... fieldChangeUpdate) {
     }
     return false;
   }
+
+  private EntityInterface getEntity(ChangeEvent event) throws IOException {
+    Class<? extends EntityInterface> entityClass = Entity.getEntityClassFromType(event.getEntityType());
+    EntityInterface entity;
+    if (event.getEntity() instanceof String) {
+      entity = JsonUtils.readValue((String) event.getEntity(), entityClass);
```

Comment:
When is getEntity a string?

Reply:
{"id":"a4e0ce34-5474-4073-a424-2a2814dc1dea","name":"dim.api/client","fullyQualifiedName":"sample_data.ecommerce_db.shopify.\"dim.api/client\"","description":"This dimension table contains a row for each channel or app that your customers use to create orders. Some examples of these include Facebook and Online Store. You can join this table with the sales table to measure channel performance.","version":0.3,"updatedAt":1673591844210,"updatedBy":"admin","href":"http://localhost:8585/api/v1/tables/a4e0ce34-5474-4073-a424-2a2814dc1dea","tableType":"Regular","columns":[{"name":"api_client_id","dataType":"NUMERIC","dataTypeDisplay":"numeric","description":"ID of the API client that called the Shopify API. For example, the ID for the online store is 580111.as","fullyQualifiedName":"sample_data.ecommerce_db.shopify.\"dim.api/client\".api_client_id","tags":[],"ordinalPosition":1},{"name":"title","dataType":"VARCHAR","dataLength":100,"dataTypeDisplay":"varchar","description":"Full name of the app or channel. For example, Point of Sale, Online Store.","fullyQualifiedName":"sample_data.ecommerce_db.shopify.\"dim.api/client\".title","tags":[],"ordinalPosition":2}],"owner":{"id":"a6a01aed-01e7-41ff-bc31-0f2b5df74eb2","type":"user","name":"admin","fullyQualifiedName":"admin","deleted":false,"href":"http://localhost:8585/api/v1/users/a6a01aed-01e7-41ff-bc31-0f2b5df74eb2"},"databaseSchema":{"id":"9f61a2b9-6902-4401-90a3-aa9c01aa2d64","type":"databaseSchema","name":"shopify","fullyQualifiedName":"sample_data.ecommerce_db.shopify","description":"This **mock** database contains schema related to shopify sales and orders with related dimension tables.","deleted":false,"href":"http://localhost:8585/api/v1/databaseSchemas/9f61a2b9-6902-4401-90a3-aa9c01aa2d64"},"database":{"id":"2f769921-c40b-46f5-98d6-9ffcfe547e43","type":"database","name":"ecommerce_db","fullyQualifiedName":"sample_data.ecommerce_db","description":"This **mock** database contains schemas related to shopify sales and orders with related dimension tables.","deleted":false,"href":"http://localhost:8585/api/v1/databases/2f769921-c40b-46f5-98d6-9ffcfe547e43"},"service":{"id":"eaaa44c8-bcf0-4e12-8d83-ea3a93071982","type":"databaseService","name":"sample_data","fullyQualifiedName":"sample_data","deleted":false,"href":"http://localhost:8585/api/v1/services/databaseServices/eaaa44c8-bcf0-4e12-8d83-ea3a93071982"},"serviceType":"BigQuery","tags":[],"changeDescription":{"fieldsAdded":[],"fieldsUpdated":[{"name":"columns.api_client_id.description","oldValue":"ID of the API client that called the Shopify API. For example, the ID for the online store is 580111.","newValue":"ID of the API client that called the Shopify API. For example, the ID for the online store is 580111.as"}],"fieldsDeleted":[],"previousVersion":0.2},"deleted":false}


Adding matchAnyOwnerName on activity feed , the above in change event comes as String instead of object